controllers/k8ssandracluster_controller.go

```diff
@@ -284,46 +292,25 @@ func (r *K8ssandraClusterReconciler) reconcileStargate(
 	return ctrl.Result{}, nil
 }
 
-func newDatacenter(k8ssandraKey types.NamespacedName, cluster string, dcNames []string, template api.CassandraDatacenterTemplateSpec, additionalSeeds []string, systemDistributedRF int) (*cassdcapi.CassandraDatacenter, error) {
-	namespace := template.Meta.Namespace
-	if len(namespace) == 0 {
-		namespace = k8ssandraKey.Namespace
-	}
+func (r *K8ssandraClusterReconciler) resolveSeedEndpoints(ctx context.Context, dc *cassdcapi.CassandraDatacenter, remoteClient client.Client) ([]string, error) {
+	podList := &corev1.PodList{}
+	labels := client.MatchingLabels{cassdcapi.DatacenterLabel: dc.Name}
 
-	config, err := cassandra.GetMergedConfig(template.Config, dcNames, systemDistributedRF, template.ServerVersion)
+	err := remoteClient.List(ctx, podList, labels)
```

Comment:
Is my code editor playing funny or do we not have rights to do this cluster-wide (no namespace restriction)? This would require ClusterRole rights to pods.

Reply:
It's not your editor ;-) Interestingly this code works. Regardless, I will add the namespace filter.